src/main/java/org/folio/service/finance/transaction/summary/OrderTransactionSummariesService.java

```diff
@@ -42,32 +44,33 @@ public Future<Void> updateOrCreateTransactionSummary(String orderId, int number,
           return updateTransactionSummary(orderId, number, requestContext);
         }
         return createTransactionSummary(new OrderTransactionSummary().withId(orderId).withNumTransactions(number),
-            requestContext)
+          requestContext)
           .mapEmpty();
       });
   }
 
   public Future<Void> createOrUpdateOrderTransactionSummary(EncumbrancesProcessingHolder holder, RequestContext requestContext) {
-      Stream<String> orderIdFromCreate = holder.getEncumbrancesForCreate()
-        .stream()
-        .map(EncumbranceRelationsHolder::getOrderId);
-      Stream<String> orderIdFromStorage = holder.getEncumbrancesFromStorage()
-        .stream()
-        .map(Transaction::getEncumbrance)
-        .map(Encumbrance::getSourcePurchaseOrderId);
-      String orderId = Stream.concat(orderIdFromCreate, orderIdFromStorage)
-        .findFirst()
-        .orElse(null);
-
-      // update or create summary if not exists
-      if (CollectionUtils.isEmpty(holder.getEncumbrancesFromStorage())) {
-        return updateOrCreateTransactionSummary(orderId, holder.getAllEncumbrancesQuantity(), requestContext);
-      } else if (holder.getAllEncumbrancesQuantity() == 0) {
-        return Future.succeededFuture();
-      } else {
-        return updateTransactionSummary(orderId, holder.getAllEncumbrancesQuantity(), requestContext);
-      }
+    Stream<String> orderIdFromCreate = holder.getEncumbrancesForCreate()
+      .stream()
+      .map(EncumbranceRelationsHolder::getOrderId);
+    Stream<String> orderIdFromStorage = holder.getEncumbrancesFromStorage()
+      .stream()
+      .map(Transaction::getEncumbrance)
+      .map(Encumbrance::getSourcePurchaseOrderId);
+    String orderId = Stream.concat(orderIdFromCreate, orderIdFromStorage)
+      .findFirst()
+      .orElse(null);
+
+    // update or create summary if not exists
+    if (CollectionUtils.isEmpty(holder.getEncumbrancesFromStorage())) {
+      return updateOrCreateTransactionSummary(orderId, holder.getAllEncumbrancesQuantity(), requestContext);
+    } else if (holder.getAllEncumbrancesQuantity() == 0) {
+      return Future.succeededFuture();
+    } else {
+      return updateTransactionSummary(orderId, holder.getAllEncumbrancesQuantity(), requestContext);
```

Comment:
please see this shortcut and reformat particular part of your code not all places

<img width="477" alt="Screenshot 2024-01-22 at 19 08 54" src="https://github.com/folio-org/mod-orders/assets/113523904/8e8201fd-9875-423d-aa33-e14db7ad9384">

Reply:
and it is better to revert changes relate to reformation of code, only keep important parts

Reply:
sure, will do